### PR TITLE
[ADD] Add module web_recipients_hide

### PR DESCRIPTION
**Pull request by cmsalmeida**
This module does not add suggested recipients list on top of message box.

### CODE REVIEW

**Comment by pedrobaeza** [comment]
Hi, thanks for the contribution.

This should go into OCA/social. There's no need to modify README.md file, it's done automatically when the module is merged.

File: web_recipients_hide/README.rst [hunk 1]
```diff
@@ -0,0 +1,4 @@
+Description
```
Comment (hbrunn) [inline web_recipients_hide/README.rst:1]: use OCA's template: https://github.com/OCA/maintainer-tools/blob/master/template/module/README.rst

File: web_recipients_hide/__init__.py [hunk 2]
```diff
@@ -0,0 +1,22 @@
+# -*- coding: utf-8 -*-
```
Comment (hbrunn) [inline web_recipients_hide/__init__.py:1]: use short headers as in https://github.com/OCA/maintainer-tools/blob/master/template/module/__init__.py

File: web_recipients_hide/static/src/js/web_recipients_hide.js [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+    #
+    ##############################################################################
+*/
+
+openerp.web_recipients_hide = function(instance){
+    var module = instance.mail //loading the namespace of the 'mail' module
+
+    module.ThreadComposeMessage.include({
+        start: function () {
+            this.options.emails_from_on_composer = false;
```
Comment (hbrunn) [inline web_recipients_hide/static/src/js/web_recipients_hide.js:31]: this breaks modularity. call `this._super`

**Comment by hbrunn** [comment]
+1 for moving this to social

File: web_recipients_hide/__openerp__.py [hunk 4]
```diff
@@ -0,0 +1,35 @@
...
+#    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+#    GNU Affero General Public License for more details.
+#
+#    You should have received a copy of the GNU Affero General Public License
+#    along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#
+##############################################################################
+{
+    'name': 'Hide recipients on send message',
+    'version': '8.0.0.1.0',
```
Comment (elicoidal) [inline web_recipients_hide/__openerp__.py:25]: 8.0.1.0.0

**Comment by pedrobaeza** [comment]
Shouldn't this be in OCA/social?

**Comment by cmsalmeida** [comment]
I will move thanks!

**Comment by pedrobaeza** [comment]
Closing and waiting the new PR in OCA/social (with other name)

**Pull request closed without merging**